nomos-services/time/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+edition = "2021"
+license = { workspace = true }
+name    = "nomos-time"
+version = "0.1.0"
+
+[dependencies]
+async-trait        = "0.1"
+cryptarchia-engine = { path = "../../consensus/cryptarchia-engine", features = ["serde", "tokio"] }
```

Comment:
Sorry I just randomly saw this, which can be changed to a workspace dependency 😄

Reply:
True, I changed a few but some may have missed during the rebase. Thanks!